app/frontend/styles/_notes.scss

```diff
@@ -1,3 +1,34 @@
+/* for __applications list__ within provider_interface */
+
+.app-application-card__status {
+  text-align: right;
+}
+
+.app-application-card__note {
+  position: relative;
+  padding: 1px 0px 1px 15px;
+  margin-top: 10px;
+  margin-bottom: 5px;
+  color: $govuk-text-colour;
+  line-height: 1.25;
+  font-weight: 400;
+  font-size: 1rem;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Is this necessary? 🤔 We don't use this rule anywhere else in the app.

Reply:
It doesn't seem to make much difference, dropping.